packages/ember-glimmer/tests/integration/content-test.js

```diff
@@ -484,6 +484,30 @@ class DynamicContentTest extends RenderingTest {
     this.assertContent('hello');
     this.assertInvariants();
   }
+
+  ['@test it can render a property on a function']() {
+    let func = () => {};
+    func.aProp = 'this is a property on a function';
+
+    this.renderPath('func.aProp', { func });
+
+    this.assertContent('this is a property on a function');
+
+    this.assertStableRerender();
+
+    this.runTask(() => set(func, 'aProp', 'still a property on a function'));
```

Comment:
Also, I don't think the `watchKey` and `watchPath` is needed for glimmer, just `tagForProperty`

Since this is a regression, it has to be back ported eventually to LTS, can you please separate out the `set(func, 'aProp', ...)` updating and the `tagForProperty` into a separate commit?   I don't mind adding that capability, just don't want to do it casually in a bug fix for an LTS regression.